src/useSsrState/__docs__/story.mdx

```diff
@@ -0,0 +1,34 @@
+import {Canvas, Meta, Story} from '@storybook/addon-docs/blocks';
+import {ImportPath} from '../../__docs__/ImportPath';
+import {Example} from './example.stories';
+
+<Meta title="State/useSsrState" component={Example}/>
+
+# useSsrState
+
+Combination of hook and context provider that allows to seamlessly, and once for all
+`@react-hookz/web` hooks, declare that application is server-rendered. All you have to do is wrap
+your application with `<SsrStateProvider>` component - it will provide static value that can be only
+changed via props and accessed over `useSsrState` hook.
+
+Every isomorphic hook that has to change it's behavior depending on environment will do it basing
+off this hook state.
+
+In case there is no upper context provider, or provider has `disabled` prop - `useSsrState` will
+yield false.
+
```

Comment:
This addresses one of my questions earlier. I like this. So basically we only have to use the context provider if we are doing SSR, otherwise we can leave it out and the isomorphic hooks will initialize with a value. I think this is reasonable - not everybody has to wrap their app in a context provider.

Reply:
Sure, onyl SSR users will be forced to use this provider.